changelog/undistributed/changelog_add_dell_os6_show_term_length_2020221206.rst

```diff
@@ -0,0 +1,6 @@
+--------------------------------------------------------------------------------
```

Comment:
no need to change. just FYI. if you add multiple parsers in 1 PR, only one changelog is OK to have listed parsers in 1 file.